frontend/src/container/SearchResult.jsx

```diff
@@ -0,0 +1,202 @@
+import React from 'react'
+import { connect } from 'react-redux'
+import { translate } from 'react-i18next'
+import { Link } from 'react-router-dom'
+import {
+  PageWrapper,
+  PageTitle,
+  PageContent,
+  ListItemWrapper,
+  displayDistanceDate,
+  IconButton,
+  BREADCRUMBS_TYPE
+} from 'tracim_frontend_lib'
+import {
+  PAGE,
+  ALL_CONTENT_TYPES
+} from '../helper.js'
+import ContentItemSearch from '../component/ContentItemSearch.jsx'
+import ContentItemHeader from '../component/Workspace/ContentItemHeader.jsx'
+import {
+  newFlashMessage,
+  setCurrentNumberPage,
+  setCurrentNumberSearchResults,
+  appendSearchResultsList,
+  setBreadcrumbs
+} from '../action-creator.sync.js'
+import { getSearchedKeywords } from '../action-creator.async.js'
+
+class searchResult extends React.Component {
+  constructor (props) {
+    super(props)
+    this.state = {
+      showArchived: 0,
+      contentTypes: ALL_CONTENT_TYPES,
+      showDeleted: 0,
+      showActive: 1
+    }
+  }
+
+  customEventReducer = ({ detail: { type, data } }) => {
+    switch (type) {
+      case 'allApp_changeLang': this.buildBreadcrumbs(); break
+    }
+  }
+
+  componentDidMount () {
+    this.buildBreadcrumbs()
+  }
+
+  findPath = (parentsList) => {
+    let parentPath = ''
+    if (parentsList.length > 0) {
+      parentPath = parentsList.reduce((acc, currentParent) => `${currentParent.label}/${acc}`, '')
+    }
+    return parentPath
+  }
+
+  putContentName = (content) => {
```

Comment:
rename the 'put' in the function name. Use something like 'get' or 'append' or 'build' or 'calculate' ...